src/components/backstage/plugins/Sidebar.js

```diff
@@ -122,39 +122,18 @@ const NpmDetailsList = ({ npmData, npmDataLoadingState }) => {
   if (npmDataLoadingState === 'error') return null;
   let inner;
 
-  if (npmDataLoadingState === 'loading') {
-    inner = (
-      <ContentLoader 
-        speed={2}
-        width={400}
-        height={240}
-        viewBox="0 0 400 240"
-        backgroundColor="#f3f3f3"
-        foregroundColor="#ecebeb"
-      >
-        <rect x="0" y="0" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="0" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="0" y="40" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="40" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="0" y="80" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="80" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="0" y="120" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="120" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="0" y="160" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="160" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="0" y="200" rx="3" ry="3" width="150" height="24" /> 
-        <rect x="220" y="200" rx="3" ry="3" width="150" height="24" /> 
```

Comment:
Same, but I also added another 2 rects to act as placeholders for the number of downloads in the past month.